app/services/xccdf_report_parser.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-# Error to raise if no metadata is available
-class EmptyMetadataError < StandardError; end
+# Error to raise if id is not available
+class MissingIdError < StandardError; end
```

Comment:
It doesn't look like we need metadata anymore - we just need to make sure the ID is there - so that's what I'm checking for now.